### ISSUE TITLE
Live region content not read by screen reader in Firefox

### ISSUE THREAD

**Issue by geo-connor** [pqina/filepond#403]
### Summary

Content added to the live region (`filepond--assistant`) is not read by screen reader (NVDA) in Firefox. This appears to be related to the `aria-relevant="additions"` attribute.

...

### How to reproduce
Run NVDA and add, upload, or remove a file from filePond in Firefox.
...

### Expected behaviour
The appropriate text is added to the `aria-live="polite"` element, and announced by the screen reader.

Currently, the text is added to the element, but the content is not announced.
...

### Additional information
When the `aria-relevent` attribute is removed from the live region element, or the attribute is set to`aria-relevant="additions text"`, Firefox *will* announce the status updates.

This appears to be due to one of several inconsistencies with `aria-relevant` across browsers. It is recommended to avoid this attribute if possible:
https://medium.com/dev-channel/why-authors-should-avoid-aria-relevant-5d3164fab1e3

A fix may be as simple as removing this attribute from the assistant element.
...

| Environment  | Version
|------------- | -----------
| OS           | Windows
| Device       | Desktop
| Browser      | Firefox 70.0
